src/main/java/uk/gov/gchq/palisade/exception/PalisadeRuntimeException.java

```diff
@@ -52,6 +54,7 @@ public PalisadeRuntimeException(final String message, final Throwable cause, fin
         this.status = status;
     }
 
+    @Generated
     public Status getStatus() {
         return status;
     }
```

Comment:
Add toString, hashCode and equals methods

Reply:
See the comment for PalisadeCheckedException.